net/modemmanager/README

```diff
@@ -0,0 +1,21 @@
+# OpenWrt ModemManager
+
+## Description
+
+Cellular modem control and connectivity
+
+Contains libqmi, libmbim and ModemManager.
+
+## Usage
+
```

Comment:
Since the README is placed in modemmanager package, "contains" sounds wrong.
I'm also wondering whether the usage example is sufficient. Only the `1)` looks like
a left-over.
Nitpick: is this markdown? then the file extension should fit.